VideoLocker/src/main/java/org/edx/mobile/social/SocialLoginDelegate.java

```diff
@@ -211,21 +213,25 @@ public ProfileModel call( ) {
 
                 // do SOCIAL LOGIN first
                 AuthResponse social = null;
+                HashMap<String, CharSequence> descParams = new HashMap<>();
+                String platformName = environment.getConfig().getPlatformName();
+                descParams.put("platform_name", environment.getConfig().getPlatformName());
+                descParams.put("platform_destination", environment.getConfig().getPlatformDestinationName());
```

Comment:
I guess if I did that, then it would it be usable as a super set for the instances that just require a platform_name so would be more broadly applicable.